apps/client/src/features/rundown/event-block/EventBlock.tsx

```diff
@@ -177,6 +178,12 @@ export default function EventBlock(props: EventBlockProps) {
           },
           { withDivider: true, label: 'Clone', icon: IoDuplicateOutline, onClick: () => actionHandler('clone') },
           { withDivider: true, label: 'Delete', icon: IoTrash, onClick: () => actionHandler('delete') },
+          {
+            withDivider: true,
+            label: 'Clear Report',
+            icon: IoTrashBin,
```

Comment:
I wonder if we want per event clear or clearing everything?

Otherwise:
how about just an X for symbol so it doesnt conflict with the delete?
Should it also be above delete?

```suggestion
            label: 'Clear report',
            icon: IoTrashBin,
```